pkg/codegen/internal/test/sdk_driver.go

```diff
@@ -137,7 +149,7 @@ type checkPackageSignature = func(t *testing.T, pwd string)
 //
 // The schema is the only piece that must be manually authored. Once the schema has been written, the expected outputs
 // can be generated by running `PULUMI_ACCEPT=true go test ./..." from the `pkg/codegen` directory.
-func TestSDKCodegen(t *testing.T, language string, genPackage GenPkgSignature, checkPackage checkPackageSignature) {
+func TestSDKCodegen(t *testing.T, opts *SDKCodegenOptions) {
```

Comment:
We should probably add an explanation on the test manifest and how to add mix-in files.

Reply:
Done, PTAL.